app/models/profile.rb

```diff
@@ -6,28 +6,23 @@ class Profile < ApplicationRecord
   validates :website_url, url: { allow_blank: true, no_local: true, schemes: %w[https http] }
   validates_with ProfileValidator
 
+  ATTRIBUTE_NAME_REGEX = /(?<attribute_name>\w+)=?/
```

Comment:
This regular expression matches both the getter and the setter but only the getter name is part of the `attribute_name` named capture since this maps to the value stored in the DB.